sycl/test-e2e/Basic/built-ins/helpers.hpp

```diff
@@ -34,14 +34,13 @@ void test(bool CheckDevice, double delta, FuncTy F, ExpectedTy Expected,
   sycl::buffer<bool, 1> SuccessBuf{1};
 
   // Make sure we don't use fp64 on devices that don't support it.
-  sycl::detail::get_elem_type_t<ExpectedTy> d(delta);
-
-  sycl::queue{}.submit([&](sycl::handler &cgh) {
+  sycl::queue q;
+  q.submit([&](sycl::handler &cgh) {
```

Comment:
Why?!.. You can just remove line 37 on the left and not change anything else at all. Not even line 53 on the right side...

Reply:
It's a matter of taste I suppose, It looks cleaner to use a variable name for the queue.

Reply:
Line 53 was a rogue change of the formatter but I've reverted it.

Reply:
> It's a matter of taste I suppose, It looks cleaner to use a variable name for the queue.

That's debatable, which is it has to go separately from the bugfix, if at all.

Reply:
> > It's a matter of taste I suppose, It looks cleaner to use a variable name for the queue.
> 
> That's debatable, which is why it has to go separately from the bugfix, if at all.

Reply:
> > It's a matter of taste I suppose, It looks cleaner to use a variable name for the queue.
> 
> That's debatable, which is it has to go separately from the bugfix, if at all.

Ok, have a look now.

Reply:
Made a few more changes to make sure that the upscaling does not happen the device does not support 64-bit floating point arithmetic. This was exposed by pre-commit tests.